### ISSUE TITLE
More Playwright tests for custom forms page

### ISSUE THREAD

**Issue by dantownsend** [piccolo-orm/piccolo_admin#303]
Related to https://github.com/piccolo-orm/piccolo_admin/issues/298

We could do with a mega form with all of the possible fields, and testing it with Playwright.

**Comment by sinisaos** [comment]
@dantownsend Now that the array fields work, basically all fields work as expected (except the datetime field, but that can easily be solved by adding `v-bind:format="property.format"` to the `NewForm.vue` `InputField` to get a datetime picker work in custom forms). By 'all possible fields' do you mean all Pydantic fields or Piccolo Admin InputFields?
Is this form enough?
```python
class CustomFormlModel(BaseModel):
    boolean_field: bool
    float_field: float
    integer_field: int
    string_field: str
    list_field: t.List[str]
    tuple_field: t.Tuple[str]
    time_field: datetime.time
    date_field: datetime.date
    datetime_field: datetime.datetime
```

**Comment by dantownsend** [comment]
@sinisaos Yeah, all possible fields is probably unrealistic as Pydantic potentially has so many different field types. I think the ones you've defined are OK, but I'd add `timedelta`.

**Comment by sinisaos** [comment]
@dantownsend Interesting, if we add `v-bind:format="property.format"` to the `NewForm.vue` `InputField` `FlatPickr` widget (datetime) and `DurationWidget` (timedelta) works on port `3000` but not on port `8000` (for PlayWright tests)?

**Comment by dantownsend** [comment]
@sinisaos On port 8000 it serves the compiled assets - so you'll have to run `npm run build` in the `admin_ui` folder.

**Comment by sinisaos** [comment]
@dantownsend Yes you are right. I forgot to build after the admin UI changes. Thanks. I can make PR if you want?
